.github/workflows/unified-runtime.yml

```diff
@@ -0,0 +1,127 @@
+name: Unified Runtime
+# Note: this is the very first version of UR workflow.
+# It was pretty much copy-pasted from UR repository.
+# Over time it will be split and merged into existing sycl workflows.
+
+on: workflow_call
```

Comment:
As mentioned above, I think we should define separate triggers here rather than making this called by the *SYCL Pre Commit on Linux* workflow.